solutions/swb-app/README.md

```diff
@@ -5,6 +5,6 @@
 # Code Coverage
 | Statements                  | Branches                | Functions                 | Lines             |
 | --------------------------- | ----------------------- | ------------------------- | ----------------- |
-| ![Statements](https://img.shields.io/badge/statements-81.94%25-yellow.svg?style=flat) | ![Branches](https://img.shields.io/badge/branches-100%25-brightgreen.svg?style=flat) | ![Functions](https://img.shields.io/badge/functions-100%25-brightgreen.svg?style=flat) | ![Lines](https://img.shields.io/badge/lines-81.94%25-yellow.svg?style=flat) |
+| ![Statements](https://img.shields.io/badge/statements-75.64%25-red.svg?style=flat) | ![Branches](https://img.shields.io/badge/branches-100%25-brightgreen.svg?style=flat) | ![Functions](https://img.shields.io/badge/functions-100%25-brightgreen.svg?style=flat) | ![Lines](https://img.shields.io/badge/lines-75.64%25-red.svg?style=flat) |
```

Comment:
This should have been mentioned before, but this is a sharp drop in test coverage. Should the zod models be put into the ignore file?

Reply:
good catch , I will add them to ignore files

Reply:
fixed